cypress/e2e/login.cy.js

```diff
@@ -0,0 +1,50 @@
+describe("User Authentication", () => {
+  describe("Login and Profile Access", () => {
+    it("allows the user to log in and access their profile", () => {
+      cy.visit("/");
+
+      // Click login button without changing the default credentials
+      cy.get(".login-btn").click();
+
+      // Wait for the redirection delay
+      cy.wait(3000); // Increased to account for the 2-second delay plus some buffer
+      cy.url().should("include", "/Login"); // URL check for redirection to the home page, I have no idea why the homepge is called login
+    });
+  });
+
+  describe('Logout', () => {
+    it('allows the user to log out', () => {
+      // Assuming the user is logged in from the previous test
+      cy.visit('/');
+      cy.get('.login-btn').click();
+      cy.get('.logout-btn').click();
+  
+      // Check if localStorage is cleared
+      cy.window().then((win) => {
+        expect(win.localStorage.length).to.equal(0);
```

Comment:
This is a flaky test. What if the app adds new properties to the Localstorage for other features.